addons/sourcemod/scripting/szf/config.sp

```diff
@@ -434,6 +437,38 @@ ArrayList Config_LoadInfectedClasses()
 	return aClasses;
 }
 
+StringMap Config_LoadWeaponIndexes()
+{
+	KeyValues kv = LoadFile(CONFIG_WEAPONINDEXES, "Weapon Indexes");
+	if (kv == null)
+		return null;
+	
+	StringMap WeaponIndexes = new StringMap();
+	if (kv.GotoFirstSubKey(false))
+	{
+		do
+		{
+			char sName[256];
+			kv.GetSectionName(sName, sizeof(sName));
+			int iOrigIndex = StringToInt(sName);
+			
+			char sValue[512];
+			kv.GetString(NULL_STRING, sValue, sizeof(sValue));
+			
+			char sValueExploded[64][8];
+			int iCount = ExplodeString(sValue, " ", sValueExploded, sizeof(sValueExploded), sizeof(sValueExploded[]));
+			
+			if (sValueExploded[0][0] != '\0')
```

Comment:
either just check `iCount` is not 0 or don't bother checking and let `for` skip it